### ISSUE TITLE
Add ack-maybe-skel-vc-grep along with ack-skel-vc-gre

### ISSUE THREAD

**Issue by joaotavora** [leoliu/ack-el#9]
Hi Leo,

I hope this project isn't abandoned. I recently discovered it and it's very well done. Thanks!

I have one small problem with it. Here's my `ack-minibuffer-setup-hook`
```
(add-hook 'ack-minibuffer-setup-hook 'ack-skel-vc-grep t)
(add-hook 'ack-minibuffer-setup-hook 'ack-yank-symbol-at-point t)
```
Obviously, the idea is to use `git grep` and then yank the symbol at point. But what if I choose a directory without a vc project? `M-x ack` will fail, because `ack-skel-vc-grep` raises an error, since it can't work under those circunstances. If it didn't raise the error though, ack would be able to gracefully fallback to ack. So now I have the trivial.

```
(defun ack-maybe-skel-vc-grep ()
  "Try insert ta template for vc grep search.
Don't do anything if that triggers an error (the previous
template stays put)"
  (interactive)
  (ignore-errors (ack-skel-vc-grep)))
``` 

in my config, which works very nicely. Is this something you would consider adding? If so I can make a PR (or take over maintenance if you've lost interest :-))

**Comment by leoliu** [comment]
Hi @joaotavora,

Thanks for the kind words. I can't believe I have been using it a dozen times a day and never found a reason to change it :( The vc-grep integration definitely needs some improvement. When it was done I was hoping someone else might try it and gave me feedback which didn't happen.

You are very welcome to take over maintenance and future development. I am glad this project falls into very capable hands. Thanks also to your contribution to Emacs from which I am benefiting everyday.

Leo.

**Comment by joaotavora** [comment]
> You are very welcome to take over maintenance and future development. I am glad this project falls into very capable hands. Thanks also to your contribution to Emacs from which I am benefiting everyday.

Thank you. I am truly happy to be trusted like this. But maybe sleep on the matter, and if you're still sure in the morning, just give me commiter access. No need to change URL or anything. That's what I did with yasnippet anyway when I handed it over to Noam Postavski.

**Comment by leoliu** [comment]
Is add you to the collaborators all that is needed? I just sent you an invite. Another option is to phase out this repo and use GNU ELPA directly for development saving the trouble of subtree merge. Feel free to do whatever is appropriate ;)

**Comment by joaotavora** [comment]
> Is add you to the collaborators all that is needed?

Mostly. Unless you don't want the GH repo to be in your name. I remember anything more I'll let you know. I don't remember if I announced the change in emacs-devel or emacs-help, or anywhere. I got GNU ELPA push access, so that's easy too. I might change the subtree scheme in to an externals scheme (it's a bit easier). Before officially taking over, let me know if you think the change in #10 is very disruptive.
